.github/workflows/main.yml

```diff
@@ -64,5 +64,16 @@ jobs:
 
       - name: Test with pytest
         run: |
-          coverage run -m pytest tests/ -k "not test_benchmark_kpi" --durations-min=1.0 --durations=400
+          coverage run -m pytest tests/ --durations-min=1.0 --durations=400
```

Comment:
@smartie2076  This is the line where I removed the code which exluded the `test_benchmark_kpi`